### ISSUE TITLE
canvas-drag-end events not thrown when distance is zero

### ISSUE THREAD

**Issue by roof12** [openseadragon/openseadragon#1459]
Hi all,

I have defined handlers for canvas-drag and canvas-drag-end. However, if I drag and release the mouse on the same pixel that I started on, the canvas-drag events are thrown but the canvas-drag-end event is not. I believe this is by design, due to this code in MouseTracker `updatePointersUp()`:

```javascript
// Drag End
if ( tracker.dragEndHandler && !updateGPoint.currentPos.equals( updateGPoint.contactPos ) ) {
...
}
```

To support my use case, I'd like to always receive the drag end event if dragging is initiated. The only work around I can see is to use canvas-press and canvas-release, but I don't want to duplicate the drag logic that MouseTracker is providing.

Is this a bug? If not, any suggestions for working around it?

Thanks!

**Comment by msalsbery** [comment]
I'm calling this a bug, and going to fix this :)

**Comment by msalsbery** [comment]
Fixed for v2.5.0

**Comment by msalsbery** [comment]
Fixed for v2.5.0 (#1872)

Feel free to reopen if issue persists!
